composer/yamls/models/bert-base.yaml

```diff
@@ -42,20 +42,11 @@ optimizer:
     eps: 1.0e-06
     weight_decay: 1.0e-5
 schedulers:
-  - warmup:
-      warmup_method: linear
-      warmup_iters: 0.06dur
-      warmup_factor: 0
-      interval: step
-  - linear_decay:
-      start_factor: 1.0
-      end_factor: 0.0
-      total_iters: 0.94dur
-      interval: step
-      verbose: false
+  - linear_decay_with_warmup:
```

Comment:
What about the other parameters like `interval` and `verbose`?

Reply:
This PR admittedly removes support for `verbose: True`, but there's nowhere in our codebase that uses that.

`interval` is replaced by a new trainer param, `use_stepwise_schedulers`, which defaults to `True`. The only schedulers in these YAMLs which used `interval: epoch` were a few multistep schedulers, where it doesn't actually matter since their LRs only logically change on epoch boundaries anyways.

Reply:
As for other params like, say, `total_iters` in this scheduler, they're no longer necessary and have been replaced by defaults.